tools/Evm/Evm/t8n/Errors/T8NException.cs

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Core.Exceptions;
+
+namespace Evm.t8n.Errors;
+
+public class T8NException : Exception, IExceptionWithExitCode
+{
+    public T8NException(Exception e, int exitCode) : base(e.Message)
```

Comment:
pass exception to base
https://learn.microsoft.com/en-us/dotnet/standard/exceptions/how-to-create-user-defined-exceptions